src/tunein.log

```diff
@@ -0,0 +1,17 @@
+INFO:urllib3.connectionpool:Starting new HTTPS connection (1): api.amazon.com
```

Comment:
Please delete the `tunein.log` from this PR. I'll do a code cleanup that will cover the _gitignore_ for this later. There just hasn't been a PR lately that would cover cleanups.